app/src/main/java/com/rockthevote/grommet/ui/eventFlow/PartnerLoginViewModel.kt

```diff
@@ -53,10 +53,16 @@ class PartnerLoginViewModel(
                 runCatching {
                     val result = rockyService.getPartnerName(partnerId.toString(), BuildConfig.VERSION_CODE.toString()).toBlocking().value()
 
+                    val response = result.response()
+
+                    if (response?.code() == 404) {
+                        throw PartnerLoginNotFoundException("404 not found")
+                    }
```

Comment:
Currently this cannot be tested because the invalid partner ID response is a 200 whereas we expect a 404. When the API is changed, this will work